libs/src/services/contracts/GovernedContractClient.js

```diff
@@ -0,0 +1,66 @@
+const web3 = require('web3')
+const ethers = require('ethers')
+const ContractClient = require('./ContractClient')
+
+/**
+ * ABI encodes argument types and values together into one encoded string
+ * @param {Array<string>} types
+ * @param {Array<string>} values
+ */
+const abiEncode = (types, values) => {
+  const abi = new ethers.utils.AbiCoder()
+  return abi.encode(types, values)
+}
+
+/**
+ * Transform a method name and its argument types into a string-composed
+ * signature, e.g. someMethod(bytes32, int32)
```

Comment:
supernit but lets update this comment to remove the empty space between `(bytes32,` and `int32)` to mirror actual signature

thanks for commenting this :)